gym_solo/core/test_obs_observations.py

```diff
@@ -174,15 +199,15 @@ def test_labels(self, mock_num_joints, mock_joint_info):
      'FR_KFE', 'FR_ANKLE', 'HL_HFE', 'HL_KFE', 'HL_ANKLE',
      'HR_HFE', 'HR_KFE', 'HR_ANKLE']
 
-    o = obs.MotorEncoder(dummy_robot_id)
-    self.assertEqual(o.labels(), ground_truth)
+    o = self.build_obs(dummy_robot_id)
+    self.assertEqual(o.labels, ground_truth)
     
   @parameterized.expand([
     ("default", False),
     ("degrees", True)
   ])
-  @mock.patch('pybullet.getJointState', autospec=True)
-  @mock.patch('pybullet.getNumJoints', autospec=True)
```

Comment:
What does removing autospec do?

Reply:
`autospec` requires that the mock has the _exact_ same method signature as the method it is patching. However, since we are using a pybullet client wrapper now, we have extra variables being passed through. Removing autospec lets mock handle everything and makes tests a bit easier.